elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -56,6 +58,10 @@
  */
 public interface ElideStandaloneSettings {
     /* Elide settings */
+
+     public final AbstractJpaTransaction.JpaTransactionCancel JTC = (m) -> { m.unwrap(Session.class).cancelQuery(); };
```

Comment:
See if these can be consolidated into a single variable using one of Java's built in Function classes.